webapp/components/textbox.jsx

```diff
@@ -92,12 +82,12 @@ export default class Textbox extends React.Component {
         }
     }
 
-    handleChange(e) {
```

Comment:
It's to avoid the need to bind the functions and tell them what `this` should be like you'll see in the constructors of other components. For some reason, React components don't always have `this` bound correctly in non-React-lifecycle methods (lifecycle methods include `render`, `componentDidMount`, etc)